generateData/csdbConnection.js

```diff
@@ -0,0 +1,23 @@
+const cassandra = require('cassandra-driver');
+
+const { distance } = cassandra.types;
+
+require('dotenv').config();
+
+const client = new cassandra.Client({
+  contactPoints: [process.env.CSCONTACTPOINT],
+  pooling: {
+    coreConnectionsPerHost: {
+      [distance.local]: 2,
+      [distance.remote]: 1,
+    },
+    maxConnectionsPerHost: {
+      [distance.local]: 32768,
+      [distance.remote]: 2000,
```

Comment:
Good find!

Reply:
Did you get this to speed up your imports to the db? It didn't really show a difference for me.

Reply:
Sorry, no, I haven't gone back to that code of mine -- it's "abandoned" for now.

Reply:
I only wrote `coreConnectionsPerHost`. Thanks!